recipes/vscode-icons/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "vscode-icons" %}
+{% set version = "11.1.0" %}
+{% set publisher = "vscode-icons-team" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/v.{{ version }}.tar.gz
+  sha256: 9b4bdc0ad6eda38ea121d9fce1cd30c8b2e911e8c7a10b29cf24040b5af602b5
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - nodejs >=12.4.0
+    - code-server >=1.40.2    
+  host:
+    - nodejs >=12.4.0
+    - code-server >=1.40.2
+  run:
+    - nodejs >=12.4.0
+    - code-server >=1.40.2
```

Comment:
The pipelines failed before I added the `build` requirements. I believe some of these requirements can be removed but I don't want to spawn a ton of CI pipelines to figure it out. Maybe someone knows better?